client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -18,7 +18,7 @@
       <span data-tmpl="text"></span>
     </span>
   </div>
-  <div id="tooltip" class="flex-center"></div>
+  <div id="tooltip" class="flex-center "></div>
```

Comment:
What's this space for?

Reply:
Thanks for the review @chappjc, the space is intended. I remember wanting to add a class there but had to use the existing `tooltip` id.